src/handlers/web3.ts

```diff
@@ -168,17 +153,16 @@ export const getProvider = ({ chainId = ChainId.mainnet }: { chainId?: number })
  * cached provider.
  * @param payload The payload, including a method and parameters, based on
  * the Ethers.js `StaticJsonRpcProvider.send` arguments.
- * @param provider The provider to use. If `null`, the current cached web3
- * provider is used.
+ * @param provider The provider to use
  * @return The response from the `StaticJsonRpcProvider.send` call.
  */
 export const sendRpcCall = async (
   payload: {
     method: string;
     params: unknown[];
   },
-  provider: StaticJsonRpcProvider | null = null
```

Comment:
in this file, most of the changes are related to removing the optional nature for `provider` in the args - now it is required to provide a `provider` so take a look at the different places that use these functions to make sure they pass in a valid `provider`